.gitattributes

```diff
@@ -0,0 +1,8 @@
+/driver-testsuite export-ignore
```

Comment:
This folder is necessary when installing Mink as a dependency of a driver, as it is a testsuite meant to be used by drivers